setup.py

```diff
@@ -84,7 +84,7 @@
         'jsonschema',
         'netcdf4',
         'numpy',
-        'psycopg2',
+        'psycopg2-binary',
```

Comment:
I think this should still depend on `psycopg2`. The binary package is available as an installation option, but, not a package that should be depended on.

By recent discussions at [psycopg2 issue 674](https://github.com/psycopg/psycopg2/issues/674), it also doesn't sound to be completely resolved.

It would be totally fine to `pip install psycopg2-binary` in the Dockerfile, but leave the dependency alone.